projects/packages/jetpack-mu-wpcom/src/features/wpcom-blocks/timeline/timeline.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Timeline Block
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom\Timeline;
+
+if ( ! defined( 'MU_WPCOM_JETPACK_TIMELINE_BLOCK' ) ) {
+	define( 'MU_WPCOM_JETPACK_TIMELINE_BLOCK', true );
+}
+
+require_once __DIR__ . '/../../../utils.php';
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	register_block_type(
+		'jetpack/timeline',
+		array(
+			'render_callback' => __NAMESPACE__ . '\load_assets',
+		)
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Load assets on frontend.
+ *
+ * @param array  $attr    Array containing the Timeline block attributes.
+ * @param string $content String containing the Timeline block content.
+ *
+ * @return string
+ */
+function load_assets( $attr, $content ) {
+	// A block's view assets will not be required in wp-admin.
+	if ( ! is_admin() ) {
+		\jetpack_mu_wpcom_enqueue_assets( 'wpcom-blocks-timeline-view', array( 'js', 'css' ) );
+	}
+
+	return $content;
+}
+
+/**
+ * Load assets on the editor.
+ */
+function enqueue_block_editor_assets() {
+	\jetpack_mu_wpcom_enqueue_assets( 'wpcom-blocks-timeline-editor', array( 'js', 'css' ) );
+}
+if ( is_admin() ) {
```

Comment:
Why did you add these conditions? It should always be `enqueue_block_editor_assets` for block scripts that register the blocks. I don't see it present before this commit. See #41055.